app/javascript/mastodon/utils/html.ts

```diff
@@ -79,10 +53,11 @@ export function htmlStringToComponents(
 
   const {
     maxDepth = 10,
-    allowedTags = DEFAULT_ALLOWED_TAGS,
+    allowedTags = AllowedTags,
     onAttribute,
     onElement,
     onText,
+    extraArgs = {} as Arg,
```

Comment:
Kind of weird that this `as Arg` is needed 🤔

Reply:
i tried `satisfies` or nothing but it says it could be instantiated as a different subtype.